src/scripts/start-stop-status

```diff
@@ -31,7 +31,7 @@ fi
 start_daemon() {
     local ts=$(date --iso-8601=second)
     echo "${ts} Starting ${SERVICE_NAME} with: ${SERVICE_COMMAND}" >${LOG_FILE}
-    STATE_DIRECTORY=${PKGVAR} ${SERVICE_COMMAND} >>${LOG_FILE} 2>&1 &
+    STATE_DIRECTORY=${PKGVAR} ${SERVICE_COMMAND} | sed '1,200p;d' >>${LOG_FILE} 2>&1 &
```

Comment:
do you need the `2>&1` first before the pipe?

Reply:
That does change the behavior, now it writes 93 lines to the file. Then it get stuck mid-line.

```
dgentry@dsm7:/$ cat  /var/packages/Tailscale/var/tailscaled.stdout.log
2023-01-19T19:43:58-08:00 Starting tailscale with: /volume1/@appstore/Tailscale/bin/tailscaled --state=/volume1/@appdata/Tailscale/tailscaled.state --socket=/volume1/@appdata/Tailscale/tailscaled.sock --port=41641 --tun=userspace-networking
...
wgengine: Reconfig: configuring userspace WireGuard config (with 1/29 peers)
magicsock: disco: node [jB1VP] d:f8349dfff17e6c4f now using 10.1.10.240:62625
Accept: ICMPv4{100.100.165.16:0 > 100.70.216.3:0} 84 icmp ok
Accept: TCP{100.100.165.16:49834 > 100.70.216.3:5000} 64 tcp ok
Accept: TCP{100.100.165.16:49835 > 100.70.216.3:5000} 64 tcp ok
Accept: TCP{100.100.165.16:49834 > 100.70.216.3:5000} 40 tcp non-syn
Accept: TCP{100.100.165.16:49839 > 100.70.216.3:5000} 40 tcp non-syn
Accept: TCP{100.100.165.16:49836 > 100.70.216.3:5000} 40 tcp non-syn
Accept: TCP{100.100.165.16:49834 > 100.70.216.3:5000} 52 tcp non-syn
Accept: TCP{100.100.165.16:49836dgentry@dsm7:/$
```

Reply:
Paging @creachadair and @raggi for sed/shell help 😂

Reply:
oh, I bet it's buffered and waiting to flush.

I bet that midline is on some 1K/4K boundary.

Try `sed -u` to turn off some buffering?

Reply:
FWIW, I confirmed that both DSM6 and DSM7 have GNU sed with -u, if that's an uncommon thing (no clue)

Reply:
`sed -u` works much better, new log lines appear in roughly real time and the file stops growing at 201 lines — one line of the "Starting tailscale ..." echo, then 200 lines from tailscaled.